nulib/meadow
# Upgrade to Phoenix 1.6, transition to using .heex templates

# Summary 
This PR upgrades Phoenix to `v1.6.0` along with the necessary dependencies, following the guide provided by Chris McCord at https://gist.github.com/chrismccord/2ab350f154235ad4a4d0f4de6decba7b

# Specific Changes in this PR
- `mix.exs` version upgrades to Phoenix and its dependencies
- `*.html.eex` templates migrated to new `*.html.heex`format

# Version bump required by the PR

See [Semantic Versioning 2.0.0](https://semver.org/) for help discerning which is required.


# Steps to Test
Run `mix.deps.get` and the relevant meadow setup mix tasks. The app should work as normal with no feature changes.

Also please let developers know if there are any special instructions to test this in the development environment. 

# :rocket: Deployment Notes

**Note** - __if you check any of these boxes go to the [(always open) `main` <- `staging` PR](https://github.com/nulib/meadow/pulls) and add detailed notes and instructions to help out others who may end up deploying your changes to production__

- Backward compatible API changes
- Backwards-incompatible API changes


# Tested/Verified
